contribs/gmf/src/directives/lidarprofile.js

```diff
@@ -0,0 +1,137 @@
+goog.provide('gmf.lidarProfileComponent');
+goog.require('gmf');
+goog.require('ol.geom.LineString');
+
+
+gmf.module.value('gmfLidarProfileTemplateUrl',
+  /**
+     * @param {!angular.JQLite} $element Element.
+     * @param {!angular.Attributes} $attrs Attributes.
+     * @return {string} Template.
+     */
+  ($element, $attrs) => {
+    const templateUrl = $attrs['gmfLidarProfileTemplateUrl'];
+    return templateUrl !== undefined ? templateUrl :
+      `${gmf.baseTemplateUrl}/lidarprofile.html`;
+  });
+
+
+/**
+ * @param {!angular.JQLite} $element Element.
+ * @param {!angular.Attributes} $attrs Attributes.
+ * @param {!function(!angular.JQLite, !angular.Attributes): string} gmfLidarProfileTemplateUrl Template function.
+ * @return {string} Template URL.
+ * @ngInject
+ */
+function gmfLidarProfileTemplateUrl($element, $attrs, gmfLidarProfileTemplateUrl) {
+  return gmfLidarProfileTemplateUrl($element, $attrs);
+}
+
+
+/**
+ * Provide a component that display a lidar profile panel.
+ * @ngdoc component
+ * @ngname gmfLidarProfile
+ */
+gmf.lidarProfileComponent = {
+  controller: 'GmfLidarProfileController',
+  bindings: {
+    'active': '=gmfLidarProfileActive',
+    'line': '=gmfLidarProfileLine',
+    'getMapFn': '&?gmfLidarProfileMap',
```

Comment:
That's not false, but that's better to use: `<?gmfLidarProfileMap` here and: `gmf-lidar-profile-map=::ctrl.map` (with the double `::` in the html)...